src/components/form-attachment/popups.vue

```diff
@@ -173,7 +173,7 @@ $popup-width: 300px;
 #form-attachment-popups-main {
   bottom: $edge-offset;
   position: fixed;
-  right: calc($edge-offset + max(50% - #{math.div($max-width-page-body, 2)}, 0px));
```

Comment:
The popup has `position: fixed`, and we want to position it near the right edge of the window. That's quite easy to do. The only reason it was more complicated before is that the Attachments page wasn't full-width. I think we were using `$max-width-page-body` before so that the popup would line up with the rest of the content of the page.